pkg/config/validate.go

```diff
@@ -148,10 +148,24 @@ func (dc DeploymentConfig) validateVars() error {
 		if !ty.IsObjectType() && !ty.IsMapType() {
 			return errors.New("vars.labels must be a map of strings")
 		}
-		for _, v := range labels.AsValueMap() {
+		for labelName, v := range labels.AsValueMap() {
```

Comment:
That makes sense, thanks for the suggestions!  I've update this PR to address both points.  I didn't think there was any need to use const strings when the `errorMessages` map was already available, but let me know if you think otherwise.

Reply:
We'll eventually move away from the giant `errorMessages` map, but as you can see it's already giant and 2 more isn't a big deal one way or the other.